fern/definition/api.yml

```diff
@@ -0,0 +1,11 @@
+name: api
+auth: ApiKey
+auth-schemes:
+  ApiKey:
+    header: X-API-Key
+    name: apiKey
+environments: 
+  Production: https://api.metriport.com/medical/v1
```

Comment:
How to deal with the fact there are diff base URLs for Medical and Devices API? One api spec per api (MAPI/DAPI)?

Reply:
We support different URLs per endpoint, i can make that change!